control.yml

```diff
@@ -1,3 +1,3 @@
 - import_playbook: "{{ playbook_dir }}/deploy-with-{{ provider | default('lago') }}.yml"
```

Comment:
I would move all playbooks (including control.yml) under `playbooks` dir

Reply:
I will be done in the following patches.

Reply:
I didn't want to do the entire re-org in one patch because it makes the code review overwhelming.